pkg/endpoint/policy.go

```diff
@@ -71,13 +71,13 @@ func (e *Endpoint) checkEgressAccess(owner Owner, dstLabels labels.LabelArray, o
 	}
 }
 
-// allowConsumer must be called with global endpoint.Mutex held
-func (e *Endpoint) allowConsumer(owner Owner, id policy.NumericIdentity) bool {
+// allowIngressIdentity must be called with global endpoint.Mutex held
+func (e *Endpoint) allowIngressIdentity(owner Owner, id policy.NumericIdentity) bool {
```

Comment:
This could probably become something more like `allowIdentityOnIngress()`, but I don't feel strongly enough that I think this should block the PR.

Reply:
Good idea. I'll change the name with my upcoming egress changes.